scrubble-client/src/ts/components/GameGrid.ts

```diff
@@ -0,0 +1,53 @@
+import { Graphics, Container } from "pixi.js";
+
+export class GameGrid extends Container {
```

Comment:
(this is just me typing it out to check for understanding) So we're taking the game grid and turning it into a class. and all the functions and things we want to do to the game grid will go here as methods like your appendGridItems()?